File: scratch/replace.js
const fs = require('fs');

function replaceSlate(filename) {
    let content = fs.readFileSync(filename, 'utf-8');
    
    // Replace className="... text-slate-[345]00 ..."
    content = content.replace(/className="([^"]*)text-slate-[345]00([^"]*)"/g, (match, before, after) => {
        let newClasses = [before.trim(), after.trim()].filter(Boolean).join(' ');
        if (newClasses) {
            return `className="${newClasses}" style={{ color: "var(--text-muted)" }}`;
        } else {
            return `style={{ color: "var(--text-muted)" }}`;
        }
    });
    
    fs.writeFileSync(filename, content, 'utf-8');
}

replaceSlate('app/components/marketing/MarketingFeatures.tsx');
replaceSlate('app/components/marketing/MarketingSocialProof.tsx');

File: scratch/replace.py
import re

def replace_slate(filename):
    with open(filename, 'r', encoding='utf-8') as f:
        content = f.read()
    
    def replacer(match):
        before = match.group(1).strip()
        after = match.group(2).strip()
        new_classes = ' '.join(filter(None, [before, after]))
        if new_classes:
            return f'className="{new_classes}" style={{{{ color: "var(--text-muted)" }}}}'
        else:
            return f'style={{{{ color: "var(--text-muted)" }}}}'
            
    # Replace className="... text-slate-[345]00 ..."
    new_content = re.sub(r'className="([^"]*)text-slate-[345]00([^"]*)"', replacer, content)
    
    with open(filename, 'w', encoding='utf-8') as f:
        f.write(new_content)

replace_slate('app/components/marketing/MarketingFeatures.tsx')
replace_slate('app/components/marketing/MarketingSocialProof.tsx')
